tests/book.test.js

```diff
@@ -0,0 +1,267 @@
+                }),
+                Book.create({
+                    title: 'How To Stop Time',
+                    ISBN: '978-0-525-52289-8',
+                    AuthorId: testAuthors[0].id,
+                    GenreId: testGenres[0].id
+                })
+            ]);
+        });
+
+        describe('POST /books', () => {
+            it('add a new book to the existing database', async () => {
+                const response = await request(app)
+                    .post('/books')
+                    .send({
+                        title: 'The Humans',
+                        ISBN: '978-0-099-12345-2',
+                        AuthorId: testAuthors[0].id,
+                        GenreId: testGenres[1].id
+                    });
+
+                const newBookRecord = await Book.findByPk(response.body.id, { raw: true });
+
+                expect(response.status).to.equal(201);
+                expect(response.body.title).to.equal('The Humans');
+                expect(response.body.ISBN).to.equal('978-0-099-12345-2');    
+
+                expect(newBookRecord.title).to.equal('The Humans');
+                expect(newBookRecord.ISBN).to.equal('978-0-099-12345-2');                
+            });
+
+            it('returns an error if the ISBN already exists', async () => {
+                const response = await request(app)
+                    .post('/books')
+                    .send({
+                        title: 'Something else',
+                        ISBN: '978-0-525-52289-8',
+                        AuthorId: testAuthors[1].id,
+                        GenreId: testGenres[0].id
+                    });
+
+                expect(response.status).to.equal(400);
+                expect(response.body.error).to.equal(errorNotUnique('ISBN'));
+            });
+        });
+
+        describe('GET /books', () => {
+            it('gets all book records', async () => {
+                const response = await request(app).get('/books');
+
+                expect(response.status).to.equal(200);
+                expect(response.body.length).to.equal(3);
+
+                response.body.forEach((book) => {
+                    const expected = testBooks.find((a) => a.id === book.id);
+
+                    expect(expected.title).to.equal(book.title);
+                    expect(expected.ISBN).to.equal(book.ISBN);
+                });
+            });
+        });
+
+        describe('GET /books/:id', () => {
+            it('gets books record by id', async () => {
+                const book = testBooks[0];
+                const response = await request(app).get(`/books/${book.id}`);
+
+                expect(response.status).to.equal(200);
+                expect(response.body.title).to.equal(book.title);
+                expect(response.body.Author.id).to.equal(book.AuthorId);
+                expect(response.body.Genre.id).to.equal(book.GenreId);
+                expect(response.body.ISBN).to.equal(book.ISBN);
```

Comment:
It would be nice to add assertions to check that the eager loading is working too. You are using eager loading instead of lazy loading because of the Sequelize associations you added in your models/index.js file. By that I mean that when you do a GET request for a particular book it also returns the relevant genre and author. You have tested that the correct ids are being returned but what about the correct author name and genre. For example to check the genre you could try ```expect(response.body.Genre.genre).to.equal(testGenres[0].genre);```
It would be great if you could look through all the test files to see where you can add assertions to check for eager loading.